quiz.js

```diff
@@ -34,8 +33,12 @@ var dingHandle = setTimeout (writeDing(), 3000);
 var words = ['short', 'medium', 'long', 'interminable', 'lengthy'];
 
 var byLength = function (A, B) {
-  // your code here
-};
+    var longest = '';
+    for (var i = 0; i <words.length; i+=1) {
```

Comment:
.sort does the iteration for you. also, the goal of this question is to return a sorted array, not the longest word of the array